sdk/tests/function.rs

```diff
@@ -4434,6 +4434,34 @@ async fn function_time_hour() -> Result<(), Error> {
 	Ok(())
 }
 
+#[tokio::test]
+async fn function_time_is_leap_year() -> Result<(), Error> {
+	let sql = r#"
+		RETURN time::is::leap_year();
+		RETURN time::is::leap_year(d"1987-06-22T08:30:45Z");
+		RETURN time::is::leap_year(d"1988-06-22T08:30:45Z");
+		RETURN d'2024-09-03T02:33:15.349397Z'.is_leap_year();
+	"#;
+	let mut test = Test::new(sql).await?;
+	//
+	let tmp = test.next()?.result?;
+	assert!(tmp.is_bool());
```

Comment:
This does not really test that `time::is::leap_year()` with no argument. Maybe generate an expected val with the same code `Datetime::default().naive_utc().date().leap_year().into()` and then `assert_eq!()` on it?